pkg/validation/internal/bundle.go

```diff
@@ -0,0 +1,96 @@
+package internal
```

Comment:
Is there a reason we want to [prevent others from using this code](https://golang.org/doc/go1.4#internalpackages)?

Reply:
They can use this code through exported `pkg/validation/validation.go` functions. We can make sub-validation functions in `pkg/validation/internal` into `Validator`s themselves and export them too.